packages/flutter/test/widgets/test_widgets.dart

```diff
@@ -58,3 +59,33 @@ class FlipWidgetState extends State<FlipWidget> {
 void flipStatefulWidget(WidgetTester tester, { bool skipOffstage = true }) {
   tester.state<FlipWidgetState>(find.byType(FlipWidget, skipOffstage: skipOffstage)).flip();
 }
+
+// Test sliver which always attempts to paint itself whether it is visible or not.
+// Use for checking if slivers which take sliver children paints optimally.
+class RenderMockSliverToBoxAdapter extends RenderSliverToBoxAdapter {
```

Comment:
We were chatting about this yesterday, why not put this in the rendering directory under a file name like `sliver_utils.dart`? We typically try to keep utilities like this organized in that way. Importing the rendering library here also kind of indicates this should be in rendering.

Reply:
Ah I discovered the `test_widgets.dart` file after chatting but yeah it makes sense that this should go in rendering. I'll fix this.